tools/azsdk-cli/Azure.Sdk.Tools.Cli/Services/Languages/PythonLanguageSpecificChecks.cs

```diff
@@ -132,17 +132,88 @@ public async Task<CLICheckResponse> UpdateSnippetsAsync(string packagePath, bool
         }
     }
 
-    public async Task<CLICheckResponse> LintCodeAsync(string packagePath, bool fixCheckErrors = false, CancellationToken cancellationToken = default)
```

Comment:
build was failing

Reply:
Im confused you should have async since you are `await` ing [here](https://github.com/Azure/azure-sdk-tools/pull/12450/files#diff-766f3ddc1406b30800268baf841353f3eb980cf6783514db5c5dff659a576da9R155) ?